client/src/containers/case-studies/list-item/component.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image';
+
+import { CaseStudy } from 'types/case-studies';
+
+import { Skeleton } from 'components/ui/skeleton';
+
+interface CaseStudiesListItemProps {
+  caseStudy?: CaseStudy;
+}
+
+const CaseStudiesListItem = ({ caseStudy }: CaseStudiesListItemProps) => {
+  return (
+    <div className="flex flex-col text-center">
+      <span className="mb-4 px-4 font-display">
+        {caseStudy ? <>{caseStudy.title}</> : <Skeleton className="h-6 w-full" />}
```

Comment:
You don't need to check if `caseStudy` exists here, you have already checked if it's fetched or not. Remove the Skeletons please.

Also, anytime you have this kind of conditions don't use ternaries in jsx. It's my preference, I know. But I think it's hard to read.